cmd/juju/cloud/add_test.go

```diff
@@ -65,6 +65,10 @@ func (f *fakeCloudMetadataStore) WritePersonalCloudMetadata(cloudsMap map[string
 
 func (f *fakeCloudMetadataStore) ParseOneCloud(data []byte) (cloudfile.Cloud, error) {
 	results := f.MethodCall(f, "ParseOneCloud", data)
+	if len(results) != 2 {
```

Comment:
This was annoying me during testing, if the data doesn't match, the test panics.  Put in for debugging, since the printed []byte is not helpful to reading strings to find what's wrong.